.circleci/config.yml

```diff
@@ -342,9 +342,11 @@ steps-mksnapshot-build: &steps-mksnapshot-build
         command: |
           cd src
           ninja -C out/native_mksnapshot v8:mksnapshot
+          ninja -C out/native_mksnapshot electron:electron_mksnapshot_zip
+          cp out/mksnapshot/mksnapshot.zip out/native_mksnapshot/mksnapshot_native.zip
     - store_artifacts:
-        path: src/out/native_mksnapshot/mksnapshot
-        destination: mksnapshot
+        path: src/out/native_mksnapshot/native_mksnapshot.zip
+        destination: native_mksnapshot
```

Comment:
What the reason to name it `native_mksnapshot ` instead of `native_mksnapshot`.
Even its archives on the Electron release page are called for example `mksnapshot-v2.0.10-linux-arm64-x64.zip`.
https://github.com/electron/electron/releases

Reply:
@alexeykuzmin for arm/arm64 we end up building two mksnapshots - one that is a x86 binary that can make arm/arm64 binaries and one that is an arm/arm64 binary (native_mksnapshot) that can be run on arm/arm64 hardware.  We need distinct names to save both binaries.